src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java

```diff
@@ -378,14 +376,21 @@ private static BlockInfoConfig parseBlockInfo(Location l, String json) {
         }
     }
 
-    private static String serializeBlockInfo(Config cfg) {
-        JsonObject json = new JsonObject();
 
-        for (String key : cfg.getKeys()) {
-            json.add(key, new JsonPrimitive(cfg.getString(key)));
+    private static String serializeBlockInfo(Config cfg) {
+        try {
+            StringWriter stringWriter = new StringWriter();
+            JsonWriter jsonWriter = new JsonWriter(stringWriter);
+            jsonWriter.setLenient(true);
+            jsonWriter.beginObject();
+            for (String key : cfg.getKeys()) {
+                jsonWriter.name(key).value(cfg.getString(key));
+            }
+            jsonWriter.endObject();
```

Comment:
What exactly is the advantage of using this right here?

Reply:
It's same as ``JsonObject#toString()``, but because we don't need the object, we can use JsonWriter directly without creating a new JsonObject.
This part of code is get called pretty often and keep showing on profiler. I need to measure it more to see if it could help with performance.

Reply:
Alright, I see. Some actual performance figures would be cool but I will mark this one as resolved nonetheless. 👍